worlds/oot/Cosmetics.py

```diff
@@ -769,7 +769,7 @@ def patch_instrument(rom, ootworld, symbols):
 
 def patch_cosmetics(ootworld, rom):
     # Use the world's slot seed for cosmetics
-    random.seed(ootworld.multiworld.per_slot_randoms[ootworld.player])
+    random.seed(ootworld.multiworld.per_slot_randoms[ootworld.player].random())
```

Comment:
```suggestion
random.seed(ootworld.random.random())
```
per_slot_randoms is deprecated now

Reply:
but does at least work for now, but I agree, this should be updated.